### DISCUSSION TITLE
Premium Theme value added

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#2159] [Q&A]
I mostly work with the Premium Theme and its megamenus and block animations, a great value right there. But since I work with it most of the time it got me to wondering - what else does Premium Theme offer that Basic Theme doesn't?

**Comment by meetmandeep** [comment]
Premium Theme includes the following:

Blocks
 - Accordion
 - Tabs
 - Counter
 - Portfolio

Design
 - Animations
 - Shapes & Dividers
 - Mega Menu
 - Sticky Headers

Template Library
 - Site Templates
 - Block Templates
 - Page Templates
 - Email Templates
